test/run_tests.py

```diff
@@ -96,68 +112,110 @@ def http_server():
         serverlog.close()
 
 
-def execute_tests(interpreter, browser, cli_options):
+def execute_tests(interpreter, browser, suite, sauceusername, saucekey):
     runner = [interpreter, '-m', 'robot.run']
```

Comment:
Not done. I agree that this is useful but did not have time to code this in today.